src/components/editor/metaform-editor.tsx

```diff
@@ -265,9 +265,12 @@ const MetaformEditor: React.FC<Props> = ({
             selected={ selected }
             onDeleteClick={ onFieldDeleteClick(sectionIndex, fieldIndex) }
           >
-            <Typography sx={{ color: "gray", margin: "5px" }}>
+            <Typography sx={{ color: "gray", margin: "0px 5px 0px 0px" }}>
               { field.required === true ? `${field.title} *` : field.title }
             </Typography>
+            <Typography sx={{ color: "red", margin: "0px, 5px 0px" }}>
+              { field.visibleIf?.field ? strings.formatString(strings.draftEditorScreen.editor.visibility.fieldConditionLabel, (field.visibleIf.field.replaceAll("-", " ")).replaceAll(".", " "), field.visibleIf.equals!) : ""}
+            </Typography>
```

Comment:
String operations should propably be done into a variable of its own instead of handling it in-line here.
e.g. `const visibleIfFieldName = field.visibleIf.field.replaceAll("-", " ")).replaceAll(".", " ");`

Reply:
This showing required field/section system is deleted from current version